lib/lightning_web/live/run_live/components.ex

```diff
@@ -225,8 +238,8 @@ defmodule LightningWeb.RunLive.Components do
     """
   end
 
-  attr :line, :string, required: true
-  attr :num, :integer, required: true
+  attr(:line, :string, required: true)
+  attr(:num, :integer, required: true)
```

Comment:
How are you formatting these files? There are rules set up in `.formatter.exs` to treat things like `attr` and in the case of our schemas to not wrap `field` in parentheses.

Can you check your VSCode settings and perhaps try and add this (if you don't already have it):

```
  "[elixir]": {
    "editor.wordBasedSuggestions": true,
    "editor.trimAutoWhitespace": true,
    "editor.defaultFormatter": "JakeBecker.elixir-ls"    <=== this
  },
```

Reply:
Done. Thanks